assets/src/application/ui/input/Button/style.scss

```diff
@@ -0,0 +1,491 @@
+/** -------------------------------------------------------------------------
+ * Event Espresso Button Stylesheet
+ * © 2020 Event Espresso
+ * ------------------------------------------------------------------------- */
+
+:root {
+	--ee-icon-button-size: 36px;
+}
+
+.ee-button.ee-button {
+	border-radius: var(--ee-border-radius-small);
+	border-style: solid;
+	border-width: 1px;
+	border-bottom: none;
+	box-shadow: var(--ee-box-shadow-tiny);
+	cursor: pointer;
+	display: inline-flex;
+	font-family: 'Segoe UI', Candara, 'Bitstream Vera Sans', 'DejaVu Sans', 'Bitstream Vera Sans', 'Trebuchet MS',
+		Verdana, 'Verdana Ref', sans-serif;
+	font-size: var(--ee-font-size-default);
+	font-weight: bold;
+	line-height: var(--ee-font-size-default);
+	margin: 0 0 var(--ee-margin-smaller);
+	outline: none !important;
+	position: relative;
+	text-decoration: none;
+	text-align: center;
+	text-shadow: var(--ee-text-shadow-inset-light);
+	top: 0;
+	transition: border-color 0.1s ease 0.1s, color 0.1s ease 0.1s !important;
+	vertical-align: middle;
+
+	&:active,
+	&:not([disabled]):active,
+	&:focus,
+	&:visited {
+		border-radius: var(--ee-border-radius-small);
+		border-style: solid;
+		border-width: 1px;
+		box-shadow: var(--ee-box-shadow-tiny);
+		cursor: pointer;
+		display: inline-block;
+		font-family: 'Segoe UI', Candara, 'Bitstream Vera Sans', 'DejaVu Sans', 'Bitstream Vera Sans', 'Trebuchet MS',
+			Verdana, 'Verdana Ref', sans-serif;
+		font-size: var(--ee-font-size-default);
+		font-weight: bold;
+		line-height: var(--ee-font-size-default);
+		margin: 0 0 var(--ee-margin-smaller);
+		outline: none !important;
+		position: relative;
+		text-decoration: none;
+		text-align: center;
+		text-shadow: var(--ee-text-shadow-inset-light);
+		top: 0;
+		transition: border-color 0.1s ease 0.1s, color 0.1s ease 0.1s !important;
+		vertical-align: middle;
+	}
+
+	&:hover {
+		&:not(:disabled):not([aria-disabled='true']):not(.is-secondary):not(.is-primary):not(.is-tertiary):not(.is-link) {
+			background-color: rgb(255, 255, 255);
+			color: rgb(25, 30, 35);
+			box-shadow: inset 0 0 0 1px #555d66, inset 0 0 0 2px #fff;
+		}
+	}
+
+	&:focus {
+		box-shadow: 0 0 0 1px var(--ee-color-primary-low-contrast), 0 2px 2px rgba(0, 0, 0, 0.6);
+		&:not(:disabled) {
+			background-color: rgb(255, 255, 255);
+			color: rgb(25, 30, 35);
+			box-shadow: inset 0 0 0 1px #6c7781, inset 0 0 0 2px #fff;
+			outline: 2px solid rgba(0, 0, 0, 0);
+		}
+	}
+
+	&:not(.ant-btn-icon-only),
+	&:active:not(.ant-btn-icon-only),
+	&:focus:not(.ant-btn-icon-only),
+	&:visited:not(.ant-btn-icon-only) {
+		// box-sizing: content-box !important; // Need to ask/ investigate this
+		// min-height: var(--ee-font-size-big); // Also this is questionable
+		// padding: var(--ee-padding-tiny) var(--ee-padding-default) var(--ee-padding-tiny) var(--ee-padding-small); // Also this is questionable
```

Comment:
these styles was causing this:

![Peek 2020-04-22 14-49](https://user-images.githubusercontent.com/1477453/79978632-e59f0e80-84a8-11ea-9b4f-6ce84ce4afd7.gif)

Reply:
those styles were not intended to be applied to icon buttons

Reply:
ok, thx